redact-wasm/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@arcjet/redact-wasm",
+  "version": "1.0.0-alpha.21",
+  "description": "Arcjet local analysis engine",
+  "license": "Apache-2.0",
+  "homepage": "https://arcjet.com",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/arcjet/arcjet-js.git",
+    "directory": "redact-wasm"
+  },
+  "bugs": {
+    "url": "https://github.com/arcjet/arcjet-js/issues",
+    "email": "support@arcjet.com"
+  },
+  "author": {
+    "name": "Arcjet",
+    "email": "support@arcjet.com",
+    "url": "https://arcjet.com"
+  },
+  "engines": {
+    "node": ">=18"
+  },
+  "type": "module",
+  "main": "./index.js",
+  "types": "./index.d.ts",
+  "exports": {
+    "edge-light": "./edge-light.js",
+    "workerd": "./workerd.js",
+    "default": "./index.js"
+  },
+  "files": [
+    "LICENSE",
+    "README.md",
+    "_virtual/",
+    "wasm/",
+    "*.js",
+    "*.d.ts",
+    "*.ts",
+    "!*.config.js"
+  ],
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "jco": "jco transpile wasm/arcjet_analyze_bindings_redact.component.wasm --no-wasi-shim --instantiation async -o wasm",
+    "build": "npm run jco; rollup --config rollup.config.js",
+    "lint": "eslint .",
+    "pretest": "npm run build",
+    "test": "NODE_OPTIONS=--experimental-vm-modules jest --passWithNoTests"
+  },
+  "sideEffects": [
+    "./wasm/arcjet_analyze_bindings_redact_bg.js"
+  ],
```

Comment:
```suggestion
```
This is old code from the wasm-bindgen days

Reply:
Worth removing it from `analyze` as well?

Reply:
Yeah. Please open a PR.

Reply:
https://github.com/arcjet/arcjet-js/pull/1444/files